src/period-calculation/generate-fixed-periods/generate-fixed-periods.nepali.spec.ts

```diff
@@ -71,52 +79,78 @@ describe('Nepali Calendar fixed period calculation', () => {
         }
         it('should add start and end dates for DAILY', () => {
             const [firstPeriod] = generateFixedPeriods({
-                periodType: 'DAILY',
+                periodType: FIXED_PERIOD_TYPES.DAILY,
                 ...date,
             })
             expect(firstPeriod.startDate).toEqual(firstPeriod.endDate)
             expect(firstPeriod.startDate).toEqual('2079-01-01')
         })
+
         it('should add start and end dates for YEARLY', () => {
             const periods = generateFixedPeriods({
-                periodType: 'YEARLY',
+                periodType: FIXED_PERIOD_TYPES.YEARLY,
                 ...date,
             }).map((p) => `${p.startDate}/${p.endDate}`)
             expect(periods[0]).toEqual('2079-01-01/2079-12-30')
             expect(periods[1]).toEqual('2078-01-01/2078-12-30')
-            expect(periods[2]).toEqual('2077-01-01/2077-12-31') // 2077 ends on the 31st not 30th
-            expect(periods[periods.length - 1]).toEqual('2070-01-01/2070-12-30')
+
+            // 2077 and 2026 end on the 31st not 30th
+            expect(periods[2]).toEqual('2077-01-01/2077-12-31')
+            expect(periods[periods.length - 1]).toEqual('2026-01-01/2026-12-31')
         })
+
         it('should add start and end dates for FYNOV', () => {
             const periods = generateFixedPeriods({
                 ...date,
-                periodType: 'FYNOV',
-            }).map((p) => `${p.startDate}/${p.endDate}`)
-            expect(periods[0]).toEqual('2079-11-01/2080-10-29')
-            expect(periods[3]).toEqual('2076-11-01/2077-10-30') // 2077 has 30 days in the 10th month
-            expect(periods[periods.length - 1]).toEqual('2070-11-01/2071-10-29')
+                periodType: FIXED_PERIOD_TYPES.FYNOV,
+            })
+
+            expect(periods[0]).toEqual(
+                expect.objectContaining({
+                    startDate: '2079-11-01',
+                    endDate: '2080-10-29',
+                })
+            )
+
+            // 2077 has 30 days in the 10th month
+            expect(periods[3]).toEqual(
+                expect.objectContaining({
+                    startDate: '2076-11-01',
+                    endDate: '2077-10-30',
+                })
+            )
+
+            expect(periods[periods.length - 1]).toEqual(
+                expect.objectContaining({
+                    startDate: '2026-11-01',
+                    endDate: '2027-10-30',
+                })
+            )
```

Comment:
I just saw this now while reviewing the changes. I could revert this for consistency reasons, just :+1: this comment and I'll do